examples/counter/src/CounterWithStoreSubscription.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { ReactElement } from 'react';
+import { ComponentBase, withStoreSubscription } from 'resub';
+import CounterStore from './Counter.store';
+
+interface ICounterSFCProps {
```

Comment:
(minor) Looks like you're using 2-space tabs for part of this file rather than 4-space.

Reply:
Ah yeah. I have my editor configured to default to whatever it detects in the file, but I made some new files so it used the default. :)

Reply:
It looks as if the example files already use spaces. I'll just convert over the one new file in resub src for now.

Reply:
...which I already did in the original commit. Looks like I'm 100% consistent with what's already there.